client/src/components/Footnote.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import styled from "styled-components";
+
+const StyledFooter = styled.div`
+  position: absolute;
+  width: 376px;
+  height: 71px;
+  left: 0;
+  bottom: 0;
+  top: 759px;
+
+  background: #f3f4fe;
```

Comment:
For styles related to colors and typography like fonts, please use the values from `theme.js`